### PR TITLE
Add Clipboard Managers collection

### PR DESCRIPTION
**Pull request by SpongeJohnSquareLennon**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created the topic or collection I'm changing.

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Clipboard managers are a great way to remember more things and are especially useful to users here on GitHub.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: collections/clipboard-managers/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+items:
+ - Slackadays/Clipboard
+ - p0deje/Maccy
+ - hluk/CopyQ
+ - TermiT/Flycut
+ - Clipy/Clipy
+display_name: Clipboard Managers
+created_by: SpongeJohnSquareLennon
```
Comment (ahpook) [inline collections/clipboard-managers/index.md:9]: The yaml needs a reference to the image or it won't display:
```suggestion
created_by: SpongeJohnSquareLennon
image: clipboard-managers.png
```

**Review by ahpook — CHANGES_REQUESTED** [review]
LGTM, please add an image reference and I will merge.

**Comment by SpongeJohnSquareLennon** [comment]
@aphook, just fixed it, thanks!

**Review by ahpook — APPROVED** [review]
LGTM, thanks!

**Pull request merged**